lib/elastic_record/connection.rb

```diff
@@ -17,6 +19,10 @@ def initialize(servers, options = {})
       self.bulk_stack         = []
     end
 
+    def elastic_version
+      @elastic_version ||= get_elastic_version
+    end
```

Comment:
Actually might not need this anymore. I think I took all the conditional checks out.